cmd/test-scripts/db_test.sh

```
@@ -0,0 +1,98 @@
+  exit 1
+fi
+
+# Remove the create_table.cql file
+rm create_table.cql
+
+# Step 2: Copy data using DSBulk with a limit of 1,000,000 entries
+echo "Copying a subset of data ($SAMPLE_SIZE entries) from the main table to the test table using DSBulk..."
+
+# Create and clean the temporary CSV directory
+mkdir -p $CSV_DIR
+rm -rf $CSV_DIR/*
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)